App.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
```

Comment:
Do you mean to replace the content of `./src/App.tsx`? `*.tsx` files should not live outside of `src/`.

Reply:
sorry theres duplicate app.tsx, theres the correct one in src as well